prompt2model/dataset_retriever/description_dataset_retriever.py

```diff
@@ -69,6 +83,15 @@ def __init__(
         self.reranking_dataset_info_file = reranking_dataset_info_file
         self.max_number_of_dataset_rows = max_number_of_dataset_rows
         self.allow_gated_datasets = allow_gated_datasets
+        self.auto_transform_data = auto_transform_data
+        self.total_num_points_to_transform = total_num_points_to_transform
+        if max_allowed_failed_transforms is None:
+            self.max_allowed_failed_transforms: int = total_num_points_to_transform // 3
```

Comment:
Similar to https://github.com/neulab/prompt2model/pull/396#discussion_r1565974839, avoid hard-coding this proportion here

Reply:
(unresolving this comment because the code hasn't changed...)

Reply:
Sorry I resolved because I made changes locally, but didn't push since I didn't want to interrupt your review